app/controllers/api/v1/api_controller.rb

```diff
@@ -35,10 +35,6 @@ def error_not_found
       # @note This method is performing both authentication and authorization.  The user suspended
       #       should be something added to the corresponding pundit policy.
       def authenticate!
-        # FeatureFlag endpoints don't require authentication because they're
-        # only used in the test environment (Cypress test FeatureFlag toggle)
-        return true if params[:controller]&.match?(%r{^api/v1/(feature_flags|instances|profile_images)$})
-
```

Comment:
Now getting rid of this, which is some small cleanup that was leftover from the larger PR that introduced V1